common/src/main/java/com/wynntils/features/user/WorldWaypointDistanceFeature.java

```diff
@@ -83,6 +84,10 @@ public void onRenderLevelPost(RenderLevelEvent.Post event) {
         float dy = (float) (location.y + 0.5 - cameraPos.y);
         float dz = (float) (location.z + 0.5 - cameraPos.z);
 
+        if (location.y <= 0 || location.y > 255) {
+            dy = (float) (cameraPos.y + 0.5 - cameraPos.y);
```

Comment:
Wait, I do not get this line. Doesn't the `cameraPos.y`s cancel out each other?

Reply:
you would be right, since delta y will be 0 if camera y is taken when calculating position-camera delta, guess my brain stop functioning when I wrote it